packages/flutter_tools/lib/src/android/gradle_utils.dart

```diff
@@ -740,3 +823,113 @@ String getGradlewFileName(Platform platform) {
     return 'gradlew';
   }
 }
+
+// List of compatible Java/Gradle versions, where javaMax versions are
```

Comment:
/// for dart doc. 

I am not sure I follow "where javaMax versions are exclusive" does that mean that there is only one?